app/src/main/java/com/haishinkit/studio/UnusedClass.kt

```diff
@@ -0,0 +1,4 @@
+package com.haishinkit.studio
+
+class UnusedClass {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.kt/blob/1a4d0f274065ad6c0d0770ad005432183d879eb6/app/src/main/java/com/haishinkit/studio/UnusedClass.kt#L3"></span>Class "UnusedClass" is never used</td>
    </tr>
  </tbody>
</table>